demo/js/index.js

```diff
@@ -86,15 +183,18 @@ window.unbxdSearch.updateConfig({
   facet: {
     facetsEl: document.getElementById("facetsWrapper"),
     selectedFacetsEl: document.getElementById("selectedFacetWrapper"),
-    rangeFacetEl: document.getElementById("rangeFacetWrapper"),
-    multiLevelFacetEl: document.getElementById("bucketedFacetWrapper"),
     applyMultipleFilters:false,
     defaultOpen:"FIRST",
-    facetMultiSelect:false
+    facetMultiSelect:true,
+    onFacetLoad:function(facets){
+    },
+    isCollapsible:true,
+    isSearchable:true,
+    viewMore:true,
   },
   pagination: {
     type:'FIXED_PAGINATION',
-    el:document.querySelector(".unxPagination"),
+    el:document.querySelector("#paginationContainer"),
```

Comment:
is this a class exposed by the SDK? if yes then it should begin with the prefix `UNX-`

Reply:
no.. it will be  DOM element from customer